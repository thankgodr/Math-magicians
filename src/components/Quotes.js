import { paragraph } from 'txtgen';

const Quotes = () => (
    <div className="m-4">
        <p>
        {paragraph(2)}
        </p>
    </div>
);

export default Quotes;