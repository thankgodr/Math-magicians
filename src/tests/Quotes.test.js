import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

describe('Testing snapshot of quotesView', () => {
  test('Snapshot of quotes view', () => {
    const elem = renderer.create(<Quotes />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});