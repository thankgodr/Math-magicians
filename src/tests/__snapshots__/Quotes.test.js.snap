// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing snapshot of quotesView Snapshot of quotes view 1`] = `
<div
  className="m-4"
>
  <p>
    Justo urna potenti urna massa dignissimnullam sollicitudinduis diampellentesque dignissim ipsumpraesent. Diampellentesque ridiculus suspendisse leo non dictumst pellentesque inceptos maecenas interdum suspendisse molestie. Imperdiet sempellentesque porta euismod mollisfusce tinciduntsed vivamus iaculis diam duis fringilla tortornulla maecenas muspraesent platea. Mus est efficitur arcucurabitur euismod leoetiam cubilia ac dui ex. Class tinciduntsed turpisetiam volutpatvestibulum quis magnased vulputate fermentum utvestibulum arcucurabitur ligulanunc. Neque eratnulla tinciduntsed torquent accumsanpellentesque tincidunt eu mauris arcu mattis aliquam
  </p>
</div>
`;
